### PR TITLE
Hide menu bar option

### LINKED ISSUE #172: Hide menu bar option
Please, create a option to hide the menu bar.

### PR DESCRIPTION
**Pull request by viktor44**
### What is this PR

Fixes  **[Hide menu bar option #172](https://github.com/getstation/desktop-app/issues/172)**

### CODE REVIEW

**Comment by neovenise** [comment]
if this get merge I will use station on a daily basis

File: app/app-worker.ts [hunk 1]
```diff
@@ -22,10 +22,13 @@ import {
 import { dispatchUrl } from './applications/duck';
 import ManifestProvider from './applications/manifest-provider/manifest-provider';
 import DistantFetcher from './applications/manifest-provider/distant-fetcher';
+// @ts-ignore no declaration file
 import { getForeFrontNavigationStateProperty } from './applications/utils';
+// @ts-ignore no declaration file
 import { setReleaseNotesSubdockVisibility } from './auto-update/duck';
 import * as bang from './bang/duck';
 import { AlertDialogProviderServiceImpl } from './dialogs/alertDialogProvider';
+// @ts-ignore no declaration file
```
Comment (magne4000) [inline app/app-worker.ts:31]: Why do we have so many ts-ignore ?

**Review by magne4000 — COMMENTED** [review]
Nice! Really good job, I think you have touched almost every part of the app with this one.

File: app/app-worker.ts (hunk 1 above)
Comment (viktor44) [reply app/app-worker.ts:31]: > Why do we have so many ts-ignore ?

They all are *.JS files. IDE shows me the  warning like `Could not find a declaration file for module ...`

**Pull request merged**